content/zh/docs/tasks/run-application/configure-pdb.md

```diff
@@ -80,6 +80,11 @@ selector goes into the PDBs `.spec.selector`.
 在这种情况下，在控制器的 `.spec.selector` 字段中做记录，并在 PDB 的
 `.spec.selector` 字段中加入同样的选择算符。
 
+<!--
+From version 1.15 PDBs support custom controllers where the [scale subresource](/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/#scale-subresource) is enabled.
+-->
+从 1.15 版本开始，PDB 支持启用[scale 子资源](/zh/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/#scale-subresource)的自定义控制器。
```

Comment:
```suggestion
从 1.15 版本开始，PDB 支持启用
[scale 子资源](/zh/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/#scale-subresource)
的自定义控制器。
```

Reply:
thank you for review
suggestion applied